File: assignment9_2.py
# nonsmok, onepack, twopack

import pandas as pd
from scipy.stats import f_oneway

data = pd.read_excel("data2.xlsx")
a = data["nonsmok"]
b = data["onepack"]
c = data["twopack"]

print(f_oneway(a, b, c))

File: assignment9_2.R
# nonsmok, onepack, twopack

library("xlsx")

# Read excel file
df <- read.xlsx("C:/Users/admin/OneDrive/Desktop/VIT/Sem 4/Foundataion of Data Science/R and Python Codes/data22.xlsx", sheetName = "Sheet1")

One_way <- aov(Values~Group, data = df)

cat("One-way ANOVA\n")
print(summary(One_way))